File: leet_code/python/removeVowels.py
def removeVowels(S):
  vowels = "aeiou"
  newStr = ""

  i = 0
  while i < len(S):
    if S[i] not in vowels:
      newStr = newStr + S[i]
    i += 1

  return newStr

S = "leetcodeisacommunityforcoders"
removeVowels(S)

File: leet_code/javascript/removeVowels.js
const removeVowels = (S) => {
  const vowels = "aeiou"
  let newStr = ""
  for (let i = 0; i < S.length; i++) {
    if (!vowels.includes(S[i])) {
      newStr = newStr + S[i]
    }
  }
  return newStr
}

console.log(removeVowels("leetcodeisacommunityforcoders"))